docker/docker-stack-compose.yml

```diff
@@ -9,7 +9,8 @@ services:
         condition: any
     volumes:
       - /var/run/docker.sock:/var/run/docker.sock
-      - /mnt/vol/portainer/data:/data
+      - /mnt/vol/portainer:/data
+    command: --admin-password "$2y$05$5wbJ2AIVEJg6ZW8QS1c7FOPM9iQ846eb3KMku1A1twrSNaNiZRLxy"
```

Comment:
🤔 

* How was it generated?
* Is it encrypted or not?

Reply:
@neongreen https://www.notion.so/aelve/Portainer-21744362c6714acd94da1cdb83b715bb#cad1ca2b3f1e427481400fe8d92a53be

Reply:
The link to the Notion page should also be in a comment in the .yml file